redaxo/src/addons/cronjob/plugins/article_status/lib/cronjob.php

```diff
@@ -12,19 +12,26 @@ class rex_cronjob_article_status extends rex_cronjob
 {
     public function execute()
     {
-        $config = rex_plugin::get('cronjob', 'article_status')->getProperty('config');
-        $from = $config['from'];
-        $to = $config['to'];
+        $config = (array) rex_plugin::get('cronjob', 'article_status')->getProperty('config');
+        $from = (array) $config['from'];
+        $to = (array) $config['to'];
         $from['before'] = (array) $from['before'];
         $to['before'] = (array) $to['before'];
```

Comment:
```suggestion
        $config = rex_type::array(rex_plugin::get('cronjob', 'article_status')->getProperty('config'));
        $from = rex_type::array($config['from']);
        $to = rex_type::array($config['to']);
        $from['before'] = rex_type::array($from['before']);
        $to['before'] = rex_type::array($to['before']);
```

So vielleicht?
Wobei wir dann ggf. die Min-Rex-Version anheben müssen.

Reply:
schön wäre wenn man den array-shape von der 1. zeile asserten könnte, ala

```
$config = rex_type::cast(
  rex_plugin::get('cronjob', 'article_status')->getProperty('config'),
  'array{from: array{..}, to: array{..}}'
);
```

Reply:
deins geht, aber mit anderer notation. so in der art: https://github.com/redaxo/redaxo/blob/main/redaxo/src/addons/phpmailer/pages/config.php#L19-L41
Bloß halt über Aufruf von `rex_type::cast(`.

Aber generell: Warum findest du hier Casting sinnvoller als Asserting?

Reply:
sinnvoller nicht, aber in diesem fall einfacher zu lesen